_security-analytics/security.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: OpenSearch Security for Security Analytics
+nav_order: 2
+has_children: false
+---
+
+# OpenSearch Security for Security Analytics
+
+You can use OpenSearch Security with Security Analytics to assign user permissions and manage the actions that users can and cannot perform. For example, you might want one group of users to be able to create, update, or delete detectors and another group of users to only view detectors.
+
+All Security Analytics indexes are protected as system indexes. Only a super admin user or an admin user with a TLS certificate can access system indexes. For more information, see [System indexes]({{site.url}}{{site.baseurl}}/security/configuration/system-indices/).
+
+Security for Security Analytics works similarly to [Alerting security]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/security/).
+
+
+## Basic permissions
+
+As an admin user, you can use Security to assign specific permissions to users based on the specific APIs they need to access. For a list of supported APIs, see [API tools]({{site.url}}{{site.baseurl}}/security-analytics/api-tools/index/).
+
+OpenSearch Security has three built-in roles that cover most use cases for Security Analytics: `security_analytics_full_access`, `security_analytics_read_access`, and `security_analytics_ack_alerts`. For descriptions of these and other roles, see [Predefined roles]({{site.url}}{{site.baseurl}}/security/access-control/users-roles#predefined-roles).
```

Comment:
Could this be broken out into a bulleted list? Also, would it be possible to include the descriptions of those three roles in the bullet point list? The next paragraph starts with "If these roles don't meet your needs..." and the journey from the beginning of Line 21 to the beginning of Line 23 for a user is going to necessarily include visiting another page without those roles being defined here (or at least a little commentary).